modApiServer/src/org/aion/api/server/rpc3/PersonalRPCImpl.java

```diff
@@ -0,0 +1,20 @@
+package org.aion.api.server.rpc3;
+
+import java.util.Objects;
+import org.aion.crypto.ISignature;
+import org.aion.crypto.ed25519.Ed25519Signature;
+import org.aion.util.types.ByteArrayWrapper;
+
+public class PersonalRPCImpl extends PersonalRPC {
+
+    @Override
+    protected ByteArrayWrapper personal_ecRecover(String dataThatWasSigned,
+        ByteArrayWrapper signature) {
```

Comment:
If RPC server didn't check the argument null input, probably you need to check here and throw NPE

Reply:
The null checks on 'composite' objects are integrated as part of the the encode/decode process. So the params will always be checked.